variants/deploy_with_capistrano/template.rb

```diff
@@ -0,0 +1,179 @@
+puts "=" * 80
+puts "Starting variant: deploy_with_capistrano"
+puts "=" * 80
+
+append_to_file("Gemfile") do
+  <<~'EO_RUBY'
+
+    # Deployment
+    #
+    # We tell bundler not to require these gems by default because the code that
+    # requires them will explicitly require them.
+    #
+    gem "capistrano", require: false
+    gem "capistrano-bundler", require: false
+    gem "capistrano-rails", require: false
+    gem "capistrano-rbenv", require: false
+    gem "capistrano-rake", require: false
+
+    # Capistrano fails when run from Github Action CI unless these gems are
+    # present in the bundle. See https://github.com/net-ssh/net-ssh/issues/565
+    gem "ed25519", require: false
+    gem "bcrypt_pbkdf", require: false
+
+  EO_RUBY
+end
+
+run "bundle install"
+run "bundle exec cap install"
+
+old_generated_cap_config_snippet = <<~EO_RUBY
+  set :application, "my_app_name"
+  set :repo_url, "git@example.com:me/my_repo.git"
+EO_RUBY
+
+new_ackama_cap_config_snippet = <<~'EO_RUBY'
+
+  ##
+  # Uncomment and configure this block if your deployments go through a
+  # bastion/jump host.
+  #
+  # CAP_BASTION_USER = "deploy".freeze
+  # CAP_BASTION_HOST = "some-ip-or-hostname".freeze
+  # set :ssh_options, {
+  #   proxy: Net::SSH::Proxy::Command.new("ssh -o StrictHostKeyChecking=no #{CAP_BASTION_USER}@#{CAP_BASTION_HOST} -W %h:%p")
+  # }
+
+  set :application, "TODO_set_app_name"
+  set :repo_url, "TODO_set_git_repo_url"
```

Comment:
We should be able to try populating this from the `git_repo_url` config variable right?